### ISSUE TITLE
Progress bar full during entire run when using reset with tqdm notebook

### ISSUE THREAD

**Issue by Lawiss** [tqdm/tqdm#864]
- [x] I have marked all applicable categories:
    + [ ] exception-raising bug
    + [x] visual output bug
    + [ ] documentation request (i.e. "X is missing from the documentation." If instead I want to ask "how to use X?" I understand [StackOverflow#tqdm] is more appropriate)
    + [ ] new feature request
- [x] I have visited the [source website], and in particular
  read the [known issues]
- [x] I have searched through the [issue tracker] for duplicates
- [x] I have mentioned version numbers, operating system and
  environment, where applicable: Ubuntu 18.04, tqdm 4.40.2, python 3.7

Hello, when I create a new bar in a notebook without a total, then use the reset method to add a total to the bar, the bar output's as a plain cyan bar that doesn't visually update, however the remaining time and iterations count work well. Step to reproduce in a notebook :
```
import tqdm.autonotebook as tqdm
bar=tqdm.notebook.tqdm()
bar.reset(10)
for i in range(10):
    sleep(3)
    bar.update(1)
```
![image](https://user-images.githubusercontent.com/30115537/70608411-60eddb00-1c00-11ea-89de-766cee4f9759.png)

However if I fill the total directly in the class constructor it works well:
```
import tqdm.autonotebook as tqdm
bar=tqdm.tqdm(total=10)
for i in range(10):
    sleep(3)
    bar.update(1)
```
![image](https://user-images.githubusercontent.com/30115537/70608687-d2c62480-1c00-11ea-950e-426902c0f3f0.png)

Is this a normal behavior ? As I understand the reset method, these two pieces of code should give equivalent result.

**Comment by casperdcl** [comment]
the import syntax is odd. Better to use:

```
from tqdm.autonotebook import tqdm
from time import sleep

bar = tqdm()
bar.reset(10)
for i in range(10):
    sleep(3)
    bar.update(1)
```

will fix soon

**Comment by Lawiss** [comment]
Thanks for the answer @casperdcl.
Indeed odd syntax but was a test in a bigger project that used an `import tqdm`, my objective was to see if replacing this import by mine (and thus adding notebook support to the module) will work without refactoring anything else.

**Comment by casperdcl** [comment]
right... `notebook` is still technically beta (haven't actually tested all inherited functionality) but at least a `reset()` patch will be merged soon (8106506). Let me know if there are any other features which don't work well.
